docs-chef-io/content/inspec/resources/aws_api_gateway_v2_api_mapping.md

```diff
@@ -0,0 +1,113 @@
++++
+title = "aws_api_gateway_v2_api_mapping resource"
+platform = "aws"
+draft = false
+gh_repo = "inspec-aws"
+
+[menu.inspec]
+title = "aws_api_gateway_v2_api_mapping"
+identifier = "inspec/resources/aws/aws_api_gateway_v2_api_mapping resource"
+parent = "inspec/resources/aws"
++++
+
+Use the `aws_api_gateway_v2_api_mapping` InSpec audit resource to test the properties of a single specific AWS APIGatewayV2 ApiMapping.
+
+For additional information, including parameters and properties, see the [AWS documentation on AWS APIGatewayV2 ApiMapping](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigatewayv2-apimapping.html).
+
+## Installation
+
+{{% inspec_aws_install %}}
+
+## Syntax
+
+Ensure that the API mapping exists.
+
+```ruby
+describe aws_api_gateway_v2_api_mapping(api_mapping_id: 'API_MAPPING_ID', domain_name: 'DOMAIN_NAME') do
+  it { should exist }
+end
+```
+
+## Parameters
+
+`api_mapping_id` _(required)_
+: The API mapping identifier.
+
+`domain_name` _(required)_
+: The domain name.
```

Comment:
both the parameters listed here are of string or hash type?

Reply:
String